crates/prql-parser/src/lexer.rs

```diff
@@ -359,6 +359,18 @@ fn escaped_character() -> impl Parser<char, char, Error = Cheap<char>> {
                     })
                 }),
         )),
+        (just("U00").ignore_then(
+            filter(|c: &char| c.is_ascii_hexdigit())
+                .repeated()
+                .exactly(6)
```

Comment:
FWIW it might be easier to express these as just 8 characters, and then let errors fall through as with any other invalid code point, rather than restricting to a prefix of `00`...

Reply:
Yeah, maybe, I don't know.